.gitignore

```diff
@@ -2,3 +2,5 @@
 *.pyc
 data
 node_modules
+manadex
```

Comment:
That's a folder that virutalenv creates (I *think*) and as such doesn't need to be checked into source control.  Contains the activate scripts, etc...